docs/ru/docs/contributing.md

```diff
@@ -82,7 +82,7 @@ $ python -m venv env
 
     </div>
 
-Ели в терминале появится ответ, что бинарник `pip` расположен по пути `.../env/bin/pip`, значит всё в порядке. 🎉
+Если в терминале появится ответ, что бинарник `pip` расположен по пути `.../env/bin/pip`, значит всё в порядке. 🎉
```

Comment:
That's my bad. Thank you.

Reply:
Thanks @Xewus for confirming this fix here is also correct! :nerd_face: